client/src/partials/templates/dateEditor.tmpl.html

```diff
@@ -0,0 +1,40 @@
+<!-- Date editor component view implementation-->
+<div 
+	ng-switch="$ctrl.editMode"
+	class="form-group"
+	ng-class="{'has-error' : $ctrl.validationTrigger && $ctrl.form.dateEditorValue.$invalid}">
+
+	<!-- Input component implementation-->
+	<div class="input-group">
+	    <input
+	    	data-date-editor
+	    	name="dateEditorValue"
+	    	type="text"
+	    	class="form-control"	    	
+	    	uib-datepicker-popup="{{$ctrl.dateFormat}}"
+	    	ng-model="$ctrl.dateValue"
+	    	is-open="$ctrl.editMode"
+	    	min-date="$ctrl.minDate"
+	    	max-date="$ctrl.maxDate"
+	    	datepicker-options="dateOptions"
+	    	close-text="Close"
```

Comment:
One of the reasons for using `uib-datepicker` is to have the ability to translate the buttons.  Could you translate this text?